scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -26,178 +24,155 @@
 import os
 import re
 import subprocess
+import sys
 import time
 
-from distutils import spawn
 
-# pylint: disable=no-self-use
+ROOT_DIR_PATH = os.path.dirname(
+    os.path.dirname(
+        os.path.dirname(
+            os.path.dirname(__file__))))
+
+GCLOUD_MIN_VERSION = (163, 0, 0)
+GCLOUD_VERSION_REGEX = r'Google Cloud SDK (.*)'
+GCLOUD_ALPHA_REGEX = r'alpha.*'
+
+REQUIRED_APIS = [
+    {'name': 'Admin SDK',
+     'service': 'admin.googleapis.com'},
+    {'name': 'AppEngine Admin',
+     'service': 'appengine.googleapis.com'},
+    {'name': 'Cloud Resource Manager',
+     'service': 'cloudresourcemanager.googleapis.com'},
+    {'name': 'Cloud SQL',
+     'service': 'sql-component.googleapis.com'},
+    {'name': 'Cloud SQL Admin',
+     'service': 'sqladmin.googleapis.com'},
+    {'name': 'Compute Engine',
+     'service': 'compute.googleapis.com'},
+    {'name': 'Deployment Manager',
+     'service': 'deploymentmanager.googleapis.com'},
+    {'name': 'IAM',
+     'service': 'iam.googleapis.com'},
+]
+
+SERVICE_ACCT_FMT = 'forseti-{}-reader-{}'
+SERVICE_ACCT_EMAIL_FMT = '{}@{}.iam.gserviceaccount.com'
+
+ORG_IAM_ROLES = [
```

Comment:
I haven't changed the IAM policies needed, but yes, I have a followup PR for docs.